defaults/main.yml

```diff
@@ -9,108 +9,221 @@
+
+# .. envvar:: roundcube__required_php_packages [[[
+#
+# List of PHP packages required by Roundcube.
+# Refer to the `official Roundcube documentation <https://github.com/roundcube/roundcubemail/wiki/Install-Requirements>`__ for details.
+roundcube__required_php_packages:
+  # Included in the base install:
+  # - 'dom'
+  # - 'iconv'
+  # - 'openssl'
+  # - 'session'
+  # - 'sockets'
+  # - 'xml'
+
+  - 'mbstring'
+  - 'json'
+
+                                                                   # ]]]
+# .. envvar:: roundcube__optional_php_packages [[[
+#
+# List of recommended/optional PHP packages for Roundcube.
+# Refer to the `official Roundcube documentation <https://github.com/roundcube/roundcubemail/wiki/Install-Requirements>`__ for details.
+roundcube__optional_php_packages:
+  # Included in the base install:
+  # - fileinfo
+
+  - 'pspell'
+  - 'zip'
+
+                                                                   # ]]]
+# .. envvar:: roundcube__custom_php_packages [[[
+#
+# List of user defined PHP packages for Roundcube.
+roundcube__custom_php_packages: []
+
+                                                                   # ]]]
+# .. envvar:: roundcube__base_php_packages [[[
+#
+# List of base PHP packages required by Roundcube.
+roundcube__base_php_packages:
+  - '{{ roundcube__required_php_packages }}'
+  - '{{ roundcube__optional_php_packages }}'
+  - '{{ [ "crypt-gpg" ] if (ansible_distribution_release in [ "sid" ]) and ("enigma" in roundcube__plugins)
+                        else [] }}'
+  - '{{ [ "mysql" ] if (roundcube__database_map[roundcube__database].dbtype == "mysql") else [] }}'
+  - '{{ [ "net-idna2" ] if ansible_distribution_release in [ "stretch", "buster", "sid", "zesty", "artful" ]
+                        else [] }}'
+  - '{{ [ "net-smtp" ] if ansible_distribution_release in [ "stretch", "buster", "sid", "xenial", "yakkety", "zesty", "artful" ]
+                       else [] }}'
+  - '{{ [ "mail-mime" ] if ansible_distribution_release in [ "stretch", "buster", "sid", "xenial", "yakkety", "zesty", "artful" ]
+                        else [] }}'
+  - '{{ [ "pear" ] if ansible_distribution_release in [ "stretch", "buster", "sid", "xenial", "yakkety", "zesty", "artful" ]
+                   else [] }}'
+  - '{{ [ "sqlite3" ] if (roundcube__database_map[roundcube__database].dbtype == "sqlite") else [] }}'
+
+                                                                   # ]]]
+# .. envvar:: roundcube__packages [[[
+#
+# Debian Packages required for the Roundcube installation.
+roundcube__packages: [ 'curl', 'file', 'unzip' ]
+
+                                                                   # ]]]
+# .. envvar:: roundcube__extra_packages [[[
 #
-# -----------------------------------
-#   Roundcube source and deployment
-# -----------------------------------
+# List of additional Debian packages (e. g. language dictionaries) that should
+# be installed with Roundcube.
+roundcube__extra_packages: []
```

Comment:
The convention I'm using is to have the list of packages in a variable named with a prefix, ie.:
```yaml
roundcube__base_packages: [ 'curl', 'file', 'unzip' ]
```
Then the users are free to use the "normal" variation in an inventory if they wish:
```yaml
roundcube__packages: []
```